### PR TITLE
New package: libqmatrixclient-0.4.2.1

### PR DESCRIPTION
**Pull request by krkk**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3392,3 +3392,4 @@ libgaminggear.so.0 libgaminggear-0.15.1_1
 libgaminggearfx.so.0 libgaminggear-0.15.1_1
 libgaminggearwidget.so.0 libgaminggear-0.15.1_1
 libopkg.so.1 libopkg-0.4.0_1
+libQMatrixClient.so.0.4 libqmatrixclient-0.4.2.1_1
```
Comment (Hoshpak) [inline common/shlibs]: are you planning on packaging an application that depends on the library?

File: srcpkgs/libqmatrixclient/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+build_style=cmake
+configure_args="-DBUILD_SHARED_LIBS=1"
+hostmakedepends="qt5-qmake qt5-host-tools"
+makedepends="qt5-devel"
+short_desc="Qt5 library to write cross-platfrom clients for Matrix"
+maintainer="Karol Kosek <krkkx@protonmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://matrix.org/docs/projects/sdk/libqmatrixclient.html"
+distfiles="https://github.com/QMatrixClient/libqmatrixclient/archive/v${version}.tar.gz"
+checksum="0efe7050ba8b866b0bf6046a9c57779173cf3d68f546d8f082ce081e8e299bc8"
```
Comment (Hoshpak) [inline srcpkgs/libqmatrixclient/template]: checksum shouldn't be quoted.

File: common/shlibs (hunk 1 above)
Comment (krkk) [reply common/shlibs]: Yes, I'm planning to separate Quaternion with this package

File: srcpkgs/libqmatrixclient/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove usr/lib/cmake
+		vmove "usr/lib/*.so"
+	}
+}
+
+post_install() {
+	# delete example application
+	rm -rf ${DESTDIR}/usr/bin
```
Comment (Johnnynator) [inline srcpkgs/libqmatrixclient/template]: Add `-DQMATRIXCLIENT_INSTALL_EXAMPLE=OFF` to `configure_args`

**Pull request merged**